CreatubblesAPIClient/Sources/DAO/GalleryDAO.swift

```diff
@@ -124,7 +124,7 @@ public class GalleryDAO: NSObject, APIClientDAO {
     }
 
     func getMyGalleriesInBatchMode(_ completion: GalleriesBatchClosure?) -> RequestHandler {
-        let fetcher = MyGalleriesQueueBatchFetcher(requestSender: requestSender, filter: .none, completion: completion)
+        let fetcher = MyGalleriesQueueBatchFetcher(requestSender: requestSender, filter: nil, completion: completion)
```

Comment:
Don't we need to pass `owned` to fetch our galleries?

Reply:
As discussed on Slack, there are additional methods for fetching galleries owned and shared by the current. This one returns any galleries connected to the current user in any way.